Client/Frontend/NotificationSurface/NotificationSurfaceManager.swift

```diff
@@ -81,7 +87,30 @@ class NotificationSurfaceManager: NotificationSurfaceDelegate {
         }
     }
 
+    func didDismissNotification(_ userInfo: [AnyHashable: Any]) {
+        guard let messageId = userInfo[Constant.messageIdKey] as? String,
+              let message = messagingManager.messageForId(messageId)
+        else { return }
+
+        messagingManager.onMessageDismissed(message)
+    }
+
     // MARK: - Private
+    private func scheduleNotification(message: GleanPlumbMessage, notificationId: String) {
+        let userInfo = [Constant.messageIdKey: message.id]
+        notificationManager.schedule(title: message.data.title ?? "",
```

Comment:
This looks a little clunky. Is there anything we can do to provide a different default string?

Reply:
I requested a fallback string for the title. I'll update once I get it.